internal/types/accountAddress_test.go

```diff
@@ -94,10 +94,10 @@ func TestStringOutput(t *testing.T) {
 		"0x1",
 		"0xf",
 		"0x1234123412341234123412341234123412341234123412340123456789abcdef",
-		"0x0234123412341234123412341234123412341234123412340123456789abcdef",
-		"0x0034123412341234123412341234123412341234123412340123456789abcdef",
-		"0x0004123412341234123412341234123412341234123412340123456789abcdef",
-		"0x0000123412341234123412341234123412341234123412340123456789abcdef",
+		"0x234123412341234123412341234123412341234123412340123456789abcdef",
+		"0x34123412341234123412341234123412341234123412340123456789abcdef",
+		"0x4123412341234123412341234123412341234123412340123456789abcdef",
+		"0x123412341234123412341234123412341234123412340123456789abcdef",
```

Comment:
This is actually not AIP-40 compliant.

The original address format (before AIP-40) takes 0x01234... and converts it to 0x1234...

But, the new format AIP-40 will be 0x01234...

And only for single character addresses, will it be shorter (0x1, 0x2, 0x3...)